lib/sqlalchemy/sql/ddl.py

```diff
@@ -539,9 +532,10 @@ def __init__(
          .. versionadded:: 1.4.0b2
 
         """
-        super().__init__(element, if_not_exists=if_not_exists)
+        super().__init__(element)
         self.columns = [CreateColumn(column) for column in element.columns]
         self.include_foreign_key_constraints = include_foreign_key_constraints
+        self.if_not_exists = if_not_exists
```

Comment:
**mike bayer** (zzzeek) wrote:

so this part feels a little ad-hoc.   can we maybe make _CreateBase and _DropBase?  because we don't really want to have any create/drop that doesn't include this parameter right?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4067